test/e2e/README.md

```diff
@@ -149,6 +149,14 @@ manifest to the master Docker container:
 go test -v github.com/vmware-tanzu/antrea/test/e2e -provider=kind
 ```
 
+You can also run the e2e tests on an existing e2e cluster with the local Antrea
```

Comment:
The first sentence sounds a little confusing as the above paragraph also says "You can also run the e2e tests with an existing Kind cluster.", this paragraph repeated it but changed "Kind cluster" to "e2e cluster". I feel perhaps adding the image load command to the above paragraph and instructions is more clear and fits for reading's order.

Reply:
Yes that was a typo from my end. I will rephrase.